components/ui/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import { ButtonHTMLAttributes, PropsWithChildren } from "react";
+import styles from "./Button.module.css";
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  className?: string;
+}
+
+const Button = ({
+  type = "button",
+  className = "",
+  disabled = false,
+  children,
+  onClick = () => {},
+}: PropsWithChildren<ButtonProps>) => {
```

Comment:
P2:
buttonProps에서 확장하셨으니 나머지 props들도 받아서 button에 넘겨주셔야 좋을 것 같습니다 😁

```suggestion
...props
}: PropsWithChildren<ButtonProps>) => {
```